src/schema/v2/TargetSupply/__tests__/index.test.ts

```diff
@@ -0,0 +1,150 @@
+import { runQuery } from "schema/v2/test/utils"
+import { StaticPathLoader } from "lib/loaders/api/loader_interface"
+
+describe("TargetSupply", () => {
+  describe("microfunnel", () => {
+    const echoingArtistLoader = (slug) => {
```

Comment:
#minor throughout our tests we reference these mock loaders directly by name. It doesn't add much value to introduce additional information to the variable, and as soon as another dev comes upon this code they'll wonder what this was for, lacking the context of your internal thought process.

Reply:
🤷 I just disagree about naming things. I think there's value in adding context. If you'd like this changed, I welcome a followup PR.

Reply:
Yah it's super minor, but wondering, what does echoing mean in this case?

Reply:
Does `echoing` here refer to the behavior of the loader taking in a slug and then returning that same slug? Sort of a pass-through situation?

Reply:
This makes sense now! ty